docs/user-management/manage-permissions.md

```diff
@@ -0,0 +1,30 @@
+---
+sidebar_position: 4
+title: Manage Permissions
+---
+
+This section explains user permissions and how to change them for users and API keys.
+
+## Permissions
```

Comment:
This Permissions section is really more concept/reference. What about separating it from the tasks? Then when you write the API section, you can just link to it and say "Refer to Permissions for what the levels mean."